package.json

```diff
@@ -73,16 +71,17 @@
     "gulp-tsc": "^1.3.0",
     "gulp-typescript": "^3.1.6",
     "husky": "^0.13.4",
-    "jest": "^20.0.4",
-    "jest-styled-components": "^4.4.1",
+    "jest": "^21.0.0",
+    "jest-styled-components": "^4.7.0",
     "lint-staged": "^4.0.0",
     "longjohn": "^0.2.12",
     "morgan": "^1.8.1",
-    "prettier": "^1.4.4",
+    "prettier": "^1.7.4",
+    "raf": "^3.4.0",
```

Comment:
What is this dependency for? https://www.npmjs.com/package/raf

Reply:
Here's a detailed description of the issue https://github.com/facebook/jest/issues/4545

But after upgrading, I started seeing the following warning for every test 
`ReferenceError: requestAnimationFrame is not defined`.

this package is a requestAnimationFrame polyfill which we can configure jest to use to suppress the warnings. I think another option is to add the following lines to `setupJest.ts`. (Someone)[https://github.com/facebook/jest/issues/4545#issuecomment-332877412] mentioned that mocking `requestAnimationFrame` that way didn't fix the issue for them but I can try it in our setup anyway. If it works, we don't need that dependency, either way, it's a temporary fix until jest is updated with better react 16 support.

```typescript
global.requestAnimationFrame = function(callback) {
  setTimeout(callback, 0);
};
```

Reply:
Gotcha, if you can’t get the simple version to work in 1 min then the temp dependency seems fine too 👍